pkg/statistics/handle/storage/read.go

```diff
@@ -535,7 +536,14 @@ func LoadNeededHistograms(sctx sessionctx.Context, statsCache util.StatsCache, l
 			err = loadNeededIndexHistograms(sctx, statsCache, item, loadFMSketch)
 		}
 		if err != nil {
-			return err
+			statslogutil.StatsLogger().Error("load needed histogram failed",
```

Comment:
@winoros Is there any further information we can log here?

Reply:
i think is no.